pkg/mount/block/blockdev.go

```diff
@@ -498,6 +498,32 @@ func (b BlockDevices) FilterPartType(guid string) BlockDevices {
 	return b.FilterNames(names...)
 }
 
+// FilterPartLabel returns a list of BlockDev objects whose underlying block
+// device has the given parition label.
+func (b BlockDevices) FilterPartLabel(label string) BlockDevices {
+	var names []string
+	for _, device := range b {
+		table, err := device.GPTTable()
+		if err != nil {
+			continue
+		}
+		for i, part := range table.Partitions {
+			if part.IsEmpty() {
+				continue
+			}
+			if strings.ToLower(part.Name()) == strings.ToLower(label) {
```

Comment:
Can you use this function which is a bit more robust? https://golang.org/pkg/strings/#EqualFold

Reply:
Sure. I took this from the already existing similar functions. Shall I change it in FilterPartID and FilterPartType, too?

Reply:
@hugelgupf I see you authored this line originally. Was there any good reason not to use EqualFold?

Reply:
ping @hugelgupf